interceptors/logging/interceptors.go

```diff
@@ -50,64 +54,105 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
 }
 
-func (c *reporter) PostMsgSend(interface{}, error, time.Duration) {}
+// PostMsgSend logs the details of the servingObject(req/resp) that is flowing out of the rpc.
+func (c *reporter) PostMsgSend(req interface{}, err error, duration time.Duration) {
```

Comment:
This is not that easy. The problem is streaming. You got messages on every stream frame. This is not what we want probably

Reply:
Sounds fair, maybe we need to check when does the stream start and end, so that we can the log rather than streaming all the frames.
Not sure how we would check the start and end of the streaming, need to dig into the grpc docs more.